app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -94,6 +98,24 @@ public void moveItem(int fromPosition, int toPosition) {
         notifyItemMoved(fromPosition, toPosition);
     }
 
+    public void refresh() {
```

Comment:
SessionsListAdapter has no knowledge of the DbSingleton now here you are adding some more coupling.